File: Programming Basics with Python/02 Conditional statements/More exercises/01_Pipes_In_Pool.py
V = int(input())
P1 = int(input())
P2 = int(input())
H = float(input())
P1_v = P1 * H
P2_v = P2 * H
water_v = P1_v + P2_v
if V >= water_v:
    print(
        f'The pool is {(water_v / V * 100):.2f}% full. Pipe 1: {(P1_v / water_v * 100):.2f}%. Pipe 2: {(P2_v / water_v * 100):.2f}%.')
else:
    print(f'For {H} hours the pool overflows with {(water_v - V):.2f} liters.')

File: JS Basics/02 Conditional Statements - More Exercises/01. Pipes In Pool.js
function pool(volume, P1, P2, H) {
    volume = Number(volume);
    let volumePipe1 = Number(P1) * Number(H);
    let volumePipe2 = Number(P2) * Number(H);
    let totalVolume = volumePipe1 + volumePipe2;
    let percentPipe1 = volumePipe1 / totalVolume * 100;
    let percentPipe2 = volumePipe2 / totalVolume * 100;
    let percentTotal = totalVolume / volume * 100;
    if (volume >= volumePipe1 + volumePipe2) {
        console.log(`The pool is ${(percentTotal).toFixed(2)}% full.Pipe 1: ${percentPipe1.toFixed(2)}%.Pipe 2: ${percentPipe2.toFixed(2)}%.`);
    }
    else {
        console.log(`For ${H} hours the pool overflows with ${(volumePipe2 + volumePipe1 - volume).toFixed(2)} liters.`);
    }
}
pool(100, 100, 100, 2.5)